js/src/edit-block/helpers/convertToSlug.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Converts a string to a slug.
+ *
+ * @param {string} toConvert The string to convert to a slug like 'Example Here'.
+ * @return {string} The string as a slug, like 'example-here'.
+ */
+const convertToSlug = ( toConvert ) => {
+	return toConvert
```

Comment:
Not used now, but will be useful in the upcoming 'autoslugging' tickets for fields and blocks.